### ISSUE TITLE
[Expenses] Epic: Module Expenses

### ISSUE THREAD

**Issue by nielsdrost7** [InvoicePlane/InvoicePlane-v2#4]

**Comment by nielsdrost7** [comment]
## SMART Plan — Expenses module epic status

> **Context/Epic**: Expenses
>
> This is an epic/tracking issue, not a single feature — it aggregates the sub-issues below. The Measurable section tracks sub-issue completion rather than fabricating tests for the epic itself; each sub-issue gets its own SMART Plan when it's processed individually.

### Specific
Track delivery of the Expenses module: `Expense`/`ExpenseCategory`/`ExpenseItem` CRUD (done) plus planned expansions — receipt uploads and category-driven tax rate auto-fill.

### Measurable
- [x] `ExpenseResource` (list, create, edit, delete) with CRUD tests in `Modules/Expenses/Tests/Feature/ExpensesTest.php`.
- [x] `ExpenseCategoryResource` with tests in `ExpenseCategoriesTest.php`.
- [ ] #87 — Upload PDF/image receipts for expenses — not started; `Expense::attachments()` is a stubbed method that literally returns `null` (`// return $this->morphMany(Attachment::class, 'attachable'); return null;`) — there is no working file-upload field on the expense form, and no `Modules\Core\Models\Upload` records are created anywhere for expenses today. This is tracked in more detail under the dedicated #99 epic.
- [ ] #272 — Category-driven expense input (tax rate auto-fill per category) — not started; `ExpenseCategory` has no `tax_rate_id` (or similar default-tax) column, so there's nothing for the expense form to read from when a category is selected.
- [ ] Export expenses to CSV/PDF — not started; no export code (`Csv`, `Export`) found anywhere in `Modules/Expenses`.

**Definition of done for this epic**: all sub-issues above are closed and `Modules/Expenses/Tests/Feature/` covers each new capability, with the "Out of scope" list below still holding at epic close.

### Achievable
1. #87 (receipt upload) is fully tracked and scoped under the dedicated #99 epic — treat that as the source of truth for sequencing this specific piece rather than duplicating effort here.
2. #272 (tax auto-fill) is independent of #87 and only needs a `tax_rate_id` column on `ExpenseCategory` plus a form `afterStateUpdated` hook on the category select — small, unblocked.
3. CSV/PDF export has no tracked issue number yet and no existing scaffolding to build from (unlike e.g. Quotes, which at least has PDF-adjacent settings placeholders) — needs its own issue before it can be sequenced.

### Relevant
Expenses is a core input to profit/loss reporting; receipts and category-level tax handling are both everyday bookkeeping needs that are currently unimplemented, not just partially built.

### Time-bound
No fixed date — epic closes when its sub-issues close. Re-review at each sprint boundary.

### Out of scope (unchanged)
- OCR receipt scanning
- Mileage / per-diem expense types
